Gemfile

```diff
@@ -1,5 +1,6 @@
 source "https://rubygems.org"
 
 gem "concurrent-ruby", require: "concurrent"
-gem "gemstash", git: "https://github.com/bundler/gemstash.git"
+# uncomment and run `bundle install` before running tests
+# gem "gemstash", git: "https://github.com/bundler/gemstash.git"
```

Comment:
I don't think we need to comment this out. This Gemfile should be used only by people who clone the repo and want to muck with the code and run the tests. It shouldn't be part of the gem itself.